package.json

```diff
@@ -275,7 +275,7 @@
         "regenerator-runtime": "^0.14.1",
         "remarkable": "^2.0.1",
         "sanitize-html": "^1.21.1",
-        "sass": "1.34.1",
+        "sass": "1.45.0",
```

Comment:
installing buie on latest vite complains about missing `compileStringAsync`: https://sass-lang.com/documentation/js-api/functions/compilestringasync/

which isn't added until v1.45.0

Reply:
specifically this was added in the last few months: https://github.com/vitejs/vite/blob/a0ed4057c90a1135aa58d06305f446e232f63e2a/packages/vite/src/node/plugins/css.ts#L2435